File: app/src/main/java/com/kakyiretechnologies/darktheme/java/SettingsActivity.java
package com.kakyiretechnologies.darktheme.java;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.kakyiretechnologies.darktheme.R;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        //add the settings fragment to its activity
        getSupportFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();
    }
}

File: app/src/main/java/com/kakyiretechnologies/darktheme/kotlin/SettingsActivity.kt
package com.kakyiretechnologies.darktheme.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.kakyiretechnologies.darktheme.R

class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)

        //add the settings fragment to its activity
        supportFragmentManager.beginTransaction()
            .replace(android.R.id.content, SettingsFragment())
            .commit()
    }
}